Makefile

```diff
@@ -80,7 +83,7 @@ $(BUILDDIR)/:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-proto-all: proto-gen proto-lint proto-check-breaking
+proto-all: proto-lint proto-check-breaking
```

Comment:
Why remove proto-gen?

Reply:
idk! that was removed in #612 for some reason, and wasn't documented, so I'm assuming it was on accident. Added back here, along with correcting the base branch so it actually works https://github.com/celestiaorg/celestia-core/pull/737/commits/91ad715054a73b2b2ae2aa2a2d9633badecea729